service/app/services/GoogleAuthService.scala

```diff
@@ -0,0 +1,68 @@
+package services
+
+import javax.inject.Inject
+
+import domain.{User, UserRole}
+import google_auth.GoogleAuthResponse
+import hateoas.JsonApi._
+import org.apache.commons.lang3.StringUtils
+import org.joda.time.DateTime
+import play.api.Configuration
+import play.api.libs.ws.WSClient
+import repositories.UserRepository
+import services.GoogleAuthService._
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class GoogleAuthService @Inject()(ws: WSClient,
+                                  config: Configuration,
+                                  userRepository: UserRepository)(implicit ex: ExecutionContext) {
+
+  private val clientID = config.getString("google.client-id")
+
+  def authenticate(idToken: String): Future[User] =
+    validateToken(idToken).flatMap { res =>
+      userRepository.findByGoogleId(res.sub).flatMap {
+        case Some(u) => Future.successful(u)
+        case _ =>
+          val userToRegister = User(
+            username = res.email,
+            password = StringUtils.EMPTY,
+            firstName = res.given_name,
+            lastName = res.family_name,
+            role = UserRole.CUSTOMER,
+            address = None,
+            points = None,
+            buyerCategoryId = Some(1),
```

Comment:
Extract `Some(1)` to the constant. Just to be clear which buyer category you reference.